### PR TITLE
Fix pair productKindId-productKindName issue

### PR DESCRIPTION
**Pull request by darynaBackbase**
Align products.json file to match new setup: 
validproductkindandid

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
             <dependency>
                 <groupId>com.backbase.dbs</groupId>
                 <artifactId>banking-services-bom</artifactId>
-                <version>2.13.1</version>
+                <version>2.14.0-cr.200</version>
```
Comment (kwoding) [inline pom.xml]: This cr does not exist on repo, we need to find a solution for this. For now, we can use the branch.

File: src/main/resources/data/products.json [hunk 2]
```diff
@@ -1,44 +1,51 @@
...
   {
     "id": "3",
-    "productKindId": "3",
+    "productKindId": "kind3",
+    "productKindName": "Debit Card",
+    "productTypeId": "333333",
+    "productTypeName": "Debit Card"
+  },
+  {
+    "id": "4",
```
Comment (richard-blank) [inline src/main/resources/data/products.json]: threre is a correlation with the productIds in ``seed/product-group-seed.json``. If you change increment ids here you should do it there as well .

also to make it more visible the ``product-group-seed.json`` relates with these ids I would suggest to move the file one hierarchy lower from the seed folder.

File: src/main/java/com/backbase/ct/bbfuel/service/LegalEntityService.java [hunk 3]
```diff
@@ -30,8 +31,10 @@ public String ingestLegalEntity(LegalEntitiesPostRequestBody legalEntity) {
         if (response.statusCode() == SC_BAD_REQUEST &&
             response.then()
                 .extract()
-                .as(com.backbase.presentation.legalentity.rest.spec.v2.legalentities.exceptions.BadRequestException.class)
-                .getErrorCode()
+                .as(BadRequestException.class)
+                .getErrors()
+                .get(0)
+                .getMessage()
                 .matches("legalEntity.save.error.message.(.*)_ALREADY_EXISTS")) {
```
[the remark this answers is not in the document]
Comment (darynaBackbase) [reply src/main/java/com/backbase/ct/bbfuel/service/LegalEntityService.java]: Message verified - the text remains the same.

File: src/main/resources/data/products.json (hunk 2 above)
Comment (darynaBackbase) [reply src/main/resources/data/products.json]: 1. Check id correlation in product-group-seed.json - done.

2. Regarding the moving - do you suggest to move product-group-seed.json out of seed/ ?

**Pull request merged**